src/views/UserSearch/Filters.test.js

```diff
@@ -77,4 +84,16 @@ describe('Filters', () => {
     renderFilters(initialProps);
     expect(screen.getByText('ui-users.departments')).toBeInTheDocument();
   });
+
+  it('Checking presence of user types filter', () => {
+    isConsortiumEnabled.mockReturnValue(true);
+    renderFilters(initialProps);
+    expect(screen.getByText('ui-users.userType')).toBeInTheDocument();
+  });
+
+  it('Checking presence of user types filter', () => {
+    isConsortiumEnabled.mockReturnValue(false);
+    renderFilters(initialProps);
+    expect(screen.queryByText('ui-users.userType')).not.toBeInTheDocument();
+  });
```

Comment:
The tests evaluate different situations so they should have different descriptions. Maybe "shows user-types filter for consortial tenants" and "hides user-types filter for non-consortial tenants"?